recipes/py-fann2/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: py-fann2
+  version: "1.1.0"
+
+source:
```

Comment:
Please use the standard syntax from the example

Reply:
@CJ-Wright I patched the source by way of an extra git commit on my fork of their project.  Is that frowned on?
When you say to use the standard syntax, do you mean to specify the version and such as variables at the top that are reused throughout the `meta.yaml` file?

Reply:
Yes  https://conda-forge.org/docs/meta.html#build-from-tarballs-not-repos
You can put patches in the recipe. https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#patches